bundles/org.openhab.binding.tibber/src/main/java/org/openhab/binding/tibber/internal/handler/TibberHandler.java

```diff
@@ -267,6 +273,49 @@ public void getURLInput(String url) throws IOException {
         }
     }
 
+    /**
+     * Builds the {@link TimeSeries} that represents the future tibber prices.
+     * As they are valid for one hour every entry is represented by two time series entries.
+     * 
+     * @param today The prices for today
+     * @param tomorrow The prices for tomorrow.
+     * @return The {@link TimeSeries} with future values.
+     */
+    private TimeSeries buildTimeSeries(JsonArray today, JsonArray tomorrow) {
+        final TimeSeries timeSeries = new TimeSeries(TimeSeries.Policy.REPLACE);
+
+        for (JsonElement entry : today) {
+            DateTimeType startsAt = parseDateTime(entry.getAsJsonObject().get("startsAt"));
```

Comment:
It seems unneeded to create a `DateTimeType` just to have a formatted `ZonedDateTime` string parsed. Could you simply use `ZonedDateTime.parse` instead? This is what `DateTimeType` does, except it tries a little harder with different formats. 😉

Reply:
fixed.